v-next/hardhat/src/internal/builtin-plugins/network-manager/edr/utils/console-log-signatures.ts

```diff
@@ -0,0 +1,422 @@
+// ------------------------------------
```

Comment:
This PR ports more things that needed. For example, this isn't used anymore.

Reply:
This is used by the console logger that is pulled in during EDR initialization for logging.